src/host/main.js

```diff
@@ -78,41 +94,42 @@ export async function hostMain({
       });
 
       const file = plan.files.find(({ name }) => name === test.filepath);
-      const result = await agent.run(JSON.parse(textDecoder.decode(file.bufferData)));
-      if (callbackUrl) {
-        const headers = {
-          'Content-Type': 'application/json',
-          ...(callbackHeader || {}),
-        };
-        const { testId, presentationNumber, capabilities, commands } = result;
-        const body = JSON.stringify({
-          testCsvRow: testId,
-          presentationNumber,
+      const testSource = JSON.parse(textDecoder.decode(file.bufferData));
+
+      const callbackBody = {
+        testCsvRow: testSource.info?.testId,
+        presentationNumber: testSource.info?.presentationNumber,
+      };
```

Comment:
[Isn't the `info` property required?](https://github.com/w3c/aria-at/blob/fc74482e90a54bee8d372db3db5357ce49a92194/types/aria-at-file.js#L67)